docs/01-getting-started.md

```diff
@@ -1,6 +1,17 @@
 Getting started
 ===============
 
+There are 2 ways of deploying Sleeper and interacting with an instance. The first way is to deploy to AWS. If you do 
+not wish to start spending money on cloud infrastructure, the second way is to deploy to a LocalStack container on 
+your local machine. This has limited functionality but will allow you to deploy an instance, ingest some files, 
```

Comment:
Add "and will only work with small volumes of data" after "limited functionality"